.travis.yml

```diff
@@ -15,5 +15,5 @@ before_install:
   - go get github.com/stretchr/testify/assert
 
 script:
-  - go test -v -covermode=count -coverprofile=coverage.out ./nlp_string
-  - goveralls -coverprofile=coverage.out -service travis-ci -repotoken $COVERALLS_TOKEN
+  - go test -v -covermode=count -coverprofile=coverage.out ./core
```

Comment:
We have to discuss how to organize this. I think we need come up with some recursive strategy